SystemDocumentation/CodeAccess.txt

```diff
@@ -0,0 +1,61 @@
+This document will give information about our groups code for perusal and printing.
+
+To access our groups code please unzip the POS.zip file and all of the files in here will be explained below.
+
+Below this will be a brief explanation of each file.
+
+For easy instructions on printing the files in the
+root directory you need to print would be:
+Database.py, DatabaseUrlGenerator.py, DatabaseUtils.py, DbConnection.py, UserLoginPackage.py, main.py, script.py
+
+In the templates/ folder for printing:
+inventory.html, Managerpage.html, POS.html, inventory.html, transactions.html
+
+Thats our whole code base, but below are short descriptions of the files. 
+
+In the root directory of github you will find, we will explain what these files are under them. This is where the bulk of our code is.
+- Please ignore this file it just allows us to run tests for our code
+Database.py
+- This is the database class we use to access the database, please visit the system documentation for more information
+DatabaseUrlGenerator.py
+- This generates the url for our database randomly so we can change it, please visit the system documentation for more information
+DatabaseUtils.py
+- This is a file of utilities so we can keep logic out of the database class, please visit the system documentation for more information
+DbConnection.py
+- This is the class that allows us to connect to the database, please visit the system documentation for more information
+UserLoginPackage.py
+- This file allows to have a user login, please visit the system documentation for more information
+main.py
+- This is the file that connects everything there is a lot to it so, please visit the system documentation for more information
+script.py
+- These are tests for our code. They get run before our pipeline gets deployed.
+
+In the templates/ folder is all of our code for the html of the project.
```

Comment:
this may sound better if we said, `"is all of our html code for the project"` instead